PROJ.lua

```diff
@@ -21,6 +23,10 @@ files {
 }
 
 excludes {
+   "src/apps/**",
```

Comment:
For 3rdparty, since we only build what is needed, it is better to explicitly add the files needed instead of a regex. This helps a lot with upgrades and lowers the amount of files we need to compile.